src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -83,6 +86,18 @@ import KvLightbox from '~/@kiva/kv-components/vue/KvLightbox';
 const EXP_KEY = 'loan_finding_page';
 const LOAN_CARD_EXP_KEY = 'lh_new_loan_card';
 
+const spotlightData = {
```

Comment:
Can this be moved into a new file? We'll have a few of these data objects, and it would really grow this component if they live here.

Reply:
Yeah, this will be moved to a json file with this structure and new data in the ticket https://kiva.atlassian.net/browse/CORE-1017